### ISSUE TITLE
List is in wrong format, will not parse

### ISSUE THREAD

**Issue by colinstu12** [RPiList/specials#1506]
This example list is in the correct format and does successfully parse: 
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Phishing-Angriffe

This example list is in a different format and does not parse due to it containing two leading pipes and ends in a caret for each line.
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/malware

Is there a reason why the malware list has different formatting? 
Is there a version of the malware list in the correct format already?

**Comment by LizenzFass78851** [comment]
#### From the [Readme.md](https://github.com/RPiList/specials?tab=readme-ov-file#klicken-sie-dafür-bitte-hier) of the repository: 

Since the update of Pi-hole at the end of March 2023 to the version [Pi-hole FTL v5.22, Web v5.19 and Core v5.16.1](https://pi-hole.net/blog/2023/03/22/pi-hole-ftl-v5-22-web-v5-19-and-core-v5-16-1-released/), a new list format in the ABP-style format is supported  , which means that since then we have also converted some of our block lists to the ABP-style format.  To continue using our blocklists, you may need to manually update the Pi-hole using the `pihole -up` command.  Our block lists will then work again for you.

----
In short: Simply update the pihole and the lists should work again.
If you use dnsmasq (or similar), use the [separate lists for dnsmasq](https://github.com/RPiList/specials/tree/master/DNSMASQ) and similar.

**Comment by colinstu12** [comment]
@LizenzFass78851 thank you, one thing however is that I am using this list with DNSBL blocking/filtering in pfblockerng which is expecting the non-ABP-style format. 
Is this the only version of this list available?

**Comment by LizenzFass78851** [comment]
Yes.  The lists for dnsmasq are the lists that do not contain the abp style format and therefore the dnsmasq lists are also more suitable for pfsence

**Comment by RPiList** [comment]
> Is this the only version of this list available?

No. You can use our lists located in the DNSMASQ folder. Same lists, just not ABP formatted.

Greetings.

**Comment by colinstu12** [comment]
So I went to https://raw.githubusercontent.com/RPiList/specials/master/DNSMASQ/malware and it still contains the || ^ ABP formatting?

**Comment by RPiList** [comment]
> So I went to https://raw.githubusercontent.com/RPiList/specials/master/DNSMASQ/malware and it still contains the || ^ ABP formatting?

Crap. That's a mistake. Don't know why nobody of its users was pointing this out.

Will be fixed ASAP.

**Comment by RPiList** [comment]
Please try again. Should be fixed now. Sorry for all the confusion. Greetings.

**Comment by colinstu12** [comment]
> Please try again. Should be fixed now. Sorry for all the confusion. Greetings.

Works now! Perfect, TY!
